clusters/resource_cluster.go

```diff
@@ -15,6 +15,8 @@ import (
 // DefaultProvisionTimeout ...
 const DefaultProvisionTimeout = 30 * time.Minute
 
+const DbfsDeprecationWarning = "For init scripts use 'volumes', 'workspace' or cloud storage location instead of 'dbfs'."
```

Comment:
Thanks for doing this, a question - can single quotes within quotes cause an issue?

Reply:
No. We have such things in outer places as well.